### PR TITLE
refactor: renaming for lock and fine

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: src/abstract/CSBondLock.sol [hunk 1]
```diff
@@ -42,40 +42,37 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
...
         }
-        MIN_BOND_LOCK_RETENTION_PERIOD = minBondLockRetentionPeriod;
-        MAX_BOND_LOCK_RETENTION_PERIOD = maxBondLockRetentionPeriod;
+        MIN_BOND_LOCK_PERIOD = minBondLockPeriod;
+        MAX_BOND_LOCK_PERIOD = maxBondLockPeriod;
     }
 
-    /// @notice Get default bond lock retention period
-    /// @return Default bond lock retention period
-    function getBondLockRetentionPeriod() external view returns (uint256) {
```
Comment (dgusakov) [inline src/abstract/CSBondLock.sol:77]: Keep the old name also for the backward capability

File: src/abstract/CSBondLock.sol [hunk 2]
```diff
@@ -96,26 +93,26 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
         BondLock storage bondLock = _getCSBondLockStorage().bondLock[
             nodeOperatorId
         ];
-        return bondLock.retentionUntil > block.timestamp ? bondLock.amount : 0;
+        return bondLock.until > block.timestamp ? bondLock.amount : 0;
```
Comment (dgusakov) [inline src/abstract/CSBondLock.sol:96]: until -> lockUntil

File: src/abstract/CSBondLock.sol [hunk 3]
```diff
@@ -140,38 +137,33 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
     }
 
     // solhint-disable-next-line func-name-mixedcase
-    function __CSBondLock_init(
-        uint256 retentionPeriod
-    ) internal onlyInitializing {
-        _setBondLockRetentionPeriod(retentionPeriod);
+    function __CSBondLock_init(uint256 period) internal onlyInitializing {
```
Comment (dgusakov) [inline src/abstract/CSBondLock.sol:140]: lockPeriod

File: src/abstract/CSBondLock.sol (hunk 1 above)
Comment (vgorkavenko) [reply src/abstract/CSBondLock.sol:77]: Done

File: src/abstract/CSBondLock.sol (hunk 2 above)
Comment (vgorkavenko) [reply src/abstract/CSBondLock.sol]: fixed

File: src/abstract/CSBondLock.sol (hunk 3 above)
Comment (vgorkavenko) [reply src/abstract/CSBondLock.sol]: fixed

**Review by skhomuti — APPROVED** [review]

File: src/CSAccounting.sol [hunk 4]
```diff
@@ -87,18 +87,18 @@ contract CSAccounting is
     /// @param bondCurve Initial bond curve
     /// @param admin Admin role member address
     /// @param _feeDistributor Fee Distributor contract address
-    /// @param bondLockRetentionPeriod Retention period for locked bond in seconds
+    /// @param bondLockPeriod  period for locked bond in seconds
```
Comment (dgusakov) [inline src/CSAccounting.sol:90]:
```suggestion
    /// @param bondLockPeriod  Bond lock period in seconds 
```

File: src/CSModule.sol [hunk 5]
```diff
@@ -1003,7 +1003,7 @@ contract CSModule is
     }
 
     /// @notice Report EL rewards stealing for the given Node Operator
-    /// @notice The amount equal to the stolen funds plus EL stealing fine will be locked
+    /// @notice The amount equal to the stolen funds plus EL stealing additional fine will be locked
```
Comment (dgusakov) [inline src/CSModule.sol:1006]:
```suggestion
    /// @notice The final locked amount will be equal to the stolen funds plus EL stealing additional fine
```

File: src/abstract/CSBondLock.sol [hunk 6]
```diff
@@ -96,26 +112,26 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
...
         _changeBondLock({
             nodeOperatorId: nodeOperatorId,
             amount: amount,
-            retentionUntil: block.timestamp + $.bondLockRetentionPeriod
+            lockUntil: block.timestamp + $.bondLockPeriod
         });
     }
 
-    /// @dev Reduce locked bond amount for the given Node Operator without changing retention period
+    /// @dev Reduce locked bond amount for the given Node Operator without changing period
```
Comment (dgusakov) [inline src/abstract/CSBondLock.sol:134]:
```suggestion
    /// @dev Reduce the locked bond amount for the given Node Operator without changing the lock period
```

File: src/abstract/CSBondLock.sol [hunk 7]
```diff
@@ -140,38 +156,36 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
...
-        uint256 retentionPeriod
-    ) internal onlyInitializing {
-        _setBondLockRetentionPeriod(retentionPeriod);
+    function __CSBondLock_init(uint256 lockPeriod) internal onlyInitializing {
+        _setBondLockPeriod(lockPeriod);
     }
 
-    /// @dev Set default bond lock retention period. That period will be sum with the current block timestamp of lock tx
-    function _setBondLockRetentionPeriod(uint256 retentionPeriod) internal {
+    /// @dev Set default bond lock period. That period will be sum with the current block timestamp of lock tx
```
Comment (dgusakov) [inline src/abstract/CSBondLock.sol:163]:
```suggestion
    /// @dev Set default bond lock period. That period will be added to the block timestamp of the lock translation to determine the bond lock duration
```

**Review by skhomuti — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**